### PR TITLE
Add missing env to pay email service

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/services/eft_service.py [hunk 1]
```diff
@@ -181,7 +180,7 @@ def _render_email_body(cls, shortname: str, amount: str, comment: str, shortname
         env = Environment(loader=FileSystemLoader(templates_dir), autoescape=True)
         template = env.get_template('eft_refund_notification.html')
 
-        url = f"{current_app.config.get('AUTH_WEB_URL')}/pay/shortname-details/{shortname_id}"
+        url = f"{current_app.config.get('WEB_APP_URL')}pay/shortname-details/{shortname_id}"
```
Comment (Jxio) [inline pay-api/src/pay_api/services/eft_service.py:183]: WEB_APP_URL: https://dev.account.bcregistry.gov.bc.ca/

File: pay-api/devops/vaults.json [hunk 2]
```diff
@@ -34,7 +34,8 @@
         "application": [
             "postgres-pay",
             "pay-api",
-            "jwt"
+            "jwt",
+            "account-mailer"
```
Comment (seeker25) [inline pay-api/devops/vaults.json:38]: Wondering if there is any overlaps of secrets between account-mailer and pay-api... also probably better to just add it into `pay-api`

File: pay-api/src/pay_api/config.py [hunk 3]
```diff
@@ -151,7 +151,7 @@ class _Config:  # pylint: disable=too-few-public-methods
     REPORT_API_BASE_URL = f'{REPORT_API_URL + REPORT_API_VERSION}/reports'
     BCOL_API_ENDPOINT = f'{BCOL_API_URL + BCOL_API_VERSION}/'
 
-    AUTH_WEB_URL = os.getenv('AUTH_WEB_URL', '')
+    WEB_APP_URL = os.getenv('WEB_APP_URL', '')
```
Comment (seeker25) [inline pay-api/src/pay_api/config.py:154]: add this into 1pass for `pay-api`

**Comment by Jxio** [comment]
added to auth_web_url: https://dev.account.bcregistry.gov.bc.ca

**Pull request closed without merging**